cmd/egress/cmd.go

```diff
@@ -91,16 +91,19 @@ are set correctly before execution.
 			}
 
 			// Set Up Proxy
-			if config.CaCert != "" {
-				// Read in the cert file
-				cert, err := os.ReadFile(config.CaCert)
-				if err != nil {
-					fmt.Println(err)
-					os.Exit(1)
+			if !config.noTls {
+
+				if config.CaCert != "" {
```

Comment:
This is totally fine as is, but I think this could read more easily as something like this, with the warning being printed in this block as well versus later, what do you think?
```
if config.CaCert != "" && !config.noTls {
   // parse the cert logic
} else {
   fmt.Println("--cacert value will be ignored since --no-tls was passed")
}
```